notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    def calc_recall(found_indices, ground_truth):
</pre><p>Side note: It would be nice to eventually see the recall computation from raft::stats exposed through pylibraft so we can just call that. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>